Makefile

```diff
@@ -1,6 +1,9 @@
 SOURCES := $(shell find . 2>&1 | grep -E '.*\.(c|h|go)$$')
 
-.DEFAULT: ploop
+.DEFAULT: ploop.bin
 
-ploop: $(SOURCES)
-	go build -o ploop .
+ploop.bin: $(SOURCES)
+	go build -o ploop.bin .
+install: ploop.bin
+	cp ploop.sh /usr/libexec/kubernetes/kubelet-plugins/volume/exec/virtuozzo~ploop/ploop
```

Comment:
@dmission I've added ploop.sh

> To be fair, it looks ugly to me - either we do whole business in shell script or in a binary. Copy 2 files to each node instead of one - doesn't look good.

Pls, be more constructive, if you don't like something, you need to suggest something else what you think will be better. The shell script here allows to customize logging without recompiling a plugin binary. You can change a logger, you can change verbose levels, etc.